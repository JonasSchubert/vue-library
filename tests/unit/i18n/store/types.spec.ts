import {
  ActionTypes, GetterTypes, MutationTypes, RouteTypes
} from '@/i18n/store/types';

describe('ActionTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(ActionTypes).toMatchSnapshot();
  })
});

describe('GetterTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(GetterTypes).toMatchSnapshot();
  })
});

describe('MutationTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(MutationTypes).toMatchSnapshot();
  })
});

describe('RouteTypes', () => {
  test('should not change unexpected', () => {
    // Assert
    expect(RouteTypes).toMatchSnapshot();
  })
});
