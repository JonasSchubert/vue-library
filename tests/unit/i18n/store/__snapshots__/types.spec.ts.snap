// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionTypes should not change unexpected 1`] = `
Object {
  "getAllTranslations": "getAllTranslations",
  "getAvailableLocales": "getAvailableLocales",
  "getLocaleTranslations": "getLocaleTranslations",
}
`;

exports[`GetterTypes should not change unexpected 1`] = `
Object {
  "applicationToken": "applicationToken",
  "availableLocales": "availableLocales",
  "currentLocale": "currentLocale",
  "error": "error",
  "locales": "locales",
}
`;

exports[`MutationTypes should not change unexpected 1`] = `
Object {
  "setAvailableLocales": "setAvailableLocales",
  "setCurrentLocale": "setCurrentLocale",
  "setError": "setError",
  "setLocales": "setLocales",
  "updateLocales": "updateLocales",
}
`;

exports[`RouteTypes should not change unexpected 1`] = `
Object {
  "getAllTranslations": "i18n/GetAllIetfTranslations",
  "getAvailableLocales": "i18n/GetAvailableIetf",
  "getLocaleTranslations": "i18n/GetIetfTranslations",
}
`;
